.circleci/build.sh

```diff
@@ -24,30 +24,18 @@ install_pocl() {
    cd ../
 }
 
-# setup sccache wrappers
-if hash sccache 2>/dev/null; then
-    SCCACHE_BIN_DIR="/tmp/sccache"
-    mkdir -p "$SCCACHE_BIN_DIR"
-    for compiler in cc c++ gcc g++ x86_64-linux-gnu-gcc; do
-        (
-            echo "#!/bin/sh"
-            echo "exec $(which sccache) $(which $compiler) \"\$@\""
-        ) > "$SCCACHE_BIN_DIR/$compiler"
-        chmod +x "$SCCACHE_BIN_DIR/$compiler"
-    done
-    export PATH="$SCCACHE_BIN_DIR:$PATH"
-fi
-
-GLOW_DIR=$PWD
-
 # Install Glow dependencies
+sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main"
 sudo apt-get update
-sudo apt-get install -y llvm-6.0 llvm-6.0-dev libpng-dev libgoogle-glog-dev
+sudo apt-get install -y llvm-8 clang-8 llvm-8-dev libclang-8-dev libpng-dev libgoogle-glog-dev opencl-headers
```

Comment:
nit: why do we need to install opencl-headers for non open cl build?

Reply:
The DEBUG case uses openCL , installs pocl which needs these installations .
Do you think it would be better to install only for DEBUG?
Do we plan to use openCL also for non-debug cases?

Reply:
I think it's cleaner just add this into `install_pocl` function.

Reply:
> Do we plan to use openCL also for non-debug cases?

We could, but no matter what all openCL builds will invoke `install_pocl`.

Reply:
Fixed